datafusion/functions/src/datetime/to_local_time.rs

```diff
@@ -562,7 +562,7 @@ mod tests {
     fn test_to_local_time_helper(input: ScalarValue, expected: ScalarValue) {
         let res = ToLocalTimeFunc::new()
             .invoke_with_args(ScalarFunctionArgs {
-                args: &[ColumnarValue::Scalar(input)],
+                args: vec![ColumnarValue::Scalar(input)],
```

Comment:
Most of the rest of this PR is changes for the new signature

Reply:
In this case I actually changed `ScalarFunctionArgs` to get an owned `Vec`

Perhaps clippy suggested `[ColumnarValue::Scalar(input)],` instead of `&vec![ColumnarValue::Scalar(input)]` 🤔